charts/nri-prometheus/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.3.0
+appVersion: 1.4.0
```

Comment:
You don't need to change this one, the appVersion is related to the version of the application in this case is the same as the docker image tag so `1.3.0` we only have to update this when updating to a new version of the image.

Reply:
Ah, right, good catch. Fixed.